Main/Source/bodypart.cpp

```diff
@@ -814,13 +814,13 @@ leg::~leg()
 
 truth corpse::IsDestroyable(ccharacter* Char) const
 {
-  for(int c = 0; c < GetDeceased()->GetBodyParts(); ++c)
+  /*for(int c = 0; c < GetDeceased()->GetBodyParts(); ++c)
```

Comment:
Delete code instead of commenting it.  Version control can let folks get it back; not need to horde.

Reply:
OK, will do.